src/Sylius/Behat/Behaviour/ChoosesName.php

```diff
@@ -25,4 +25,18 @@ public function chooseName($name)
     {
         $this->getDocument()->selectFieldOption('Name', $name);
     }
+
+    /**
+     * If the scenario is using "@javascript" tag
+     *
+     * @param string $name
+     */
+    public function selectName($name)
```

Comment:
I would say, that is worthy to place it in separated trait. Or even in page and we will extract it if needed